commons/all/src/test/java/org/infinispan/commons/util/TimedThreadDumpTest.java

```diff
@@ -0,0 +1,19 @@
+package org.infinispan.commons.util;
+
+import org.apache.logging.log4j.Level;
+import org.apache.logging.log4j.core.config.Configurator;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class TimedThreadDumpTest {
+
+   static {
+      Configurator.setLevel("org.infinispan.commons.util", Level.TRACE);
```

Comment:
Lets set this specifically for the `org.infinispan.commons.util.TimedThreadDump` category just in case.

Reply:
Updated.